readthedocs/projects/models.py

```diff
@@ -1582,16 +1583,28 @@ class HTTPHeader(TimeStampedModel, models.Model):
     expose them all without hardcoding them.
     """
 
+    HEADERS_CHOICES = (
+        ('referrer_policy', 'Referrer-Policy'),
+        ('permissions_policy', 'Permissions-Policy'),
+        ('feature_policy', 'Feature-Policy'),
+        ('access_control_allow_origin', 'Access-Control-Allow-Origin'),
+        ('access_control_allow_headers', 'Access-Control-Allow-Headers'),
```

Comment:
Do we want users to be able to control CORS headers? These are currently controlled in our middleware, and I don't believe these will be exposed on the API without additional changes, so I'm not sure if this will actually work yet (eg. an `OPTIONS` request to `/_/api/v2/`)

Reply:
I'm not thinking of exposing these to users. I added here all the headers that I found that we are using in -ops and -corporate-ops so we can delete the custom proxito nginx files.

Reply:
Also, these CORS headers will only work on documentation pages since the middleware is executed on El Proxito only. I don't think this will affect API calls.

Reply:
Right? 😃

Reply:
I don't believe these CORS headers will be properly sent for the API though, will they? CORS doesn't matter on the documentation pages, only on reply to `OPTIONS` requests when using the API's.